diff -urN LibVNCServer-0.9.9/client_examples/gtkvncviewer.c LibVNCServer-0.9.9-patched/client_examples/gtkvncviewer.c
--- LibVNCServer-0.9.9/client_examples/gtkvncviewer.c	2012-05-05 01:19:00.000000000 +1100
+++ LibVNCServer-0.9.9-patched/client_examples/gtkvncviewer.c	2012-08-08 20:06:43.050294425 +1100
@@ -563,7 +563,7 @@
 
 	time (&log_clock);
 	strftime (buf, 255, "%d/%m/%Y %X ", localtime (&log_clock));
-	fprintf (stdout, buf);
+	fprintf (stdout, "%s", buf);
 
 	vfprintf (stdout, format, args);
 	fflush (stdout);
