--- LibVNCServer-0.9.9/configure.ac.bero	2013-01-16 00:46:33.334252055 +0100
+++ LibVNCServer-0.9.9/configure.ac	2013-01-16 00:46:40.099174247 +0100
@@ -1,8 +1,8 @@
 # Process this file with autoconf to produce a configure script.
 AC_INIT(LibVNCServer, 0.9.9, http://sourceforge.net/projects/libvncserver)
-AM_INIT_AUTOMAKE(LibVNCServer, 0.9.9)
+AM_INIT_AUTOMAKE
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
-AM_CONFIG_HEADER(rfbconfig.h)
+AC_CONFIG_HEADERS([rfbconfig.h])
 AX_PREFIX_CONFIG_H([rfb/rfbconfig.h])
 
 # Checks for programs.
